src/Adapter/Ftp.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Bolt\Filesystem\Adapter;
+
+use League\Flysystem\Adapter\Ftp as FtpAdapter;
+use League\Flysystem\Config;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+
+/**
+ * FTP adapter.
+ *
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class Ftp extends FtpAdapter
+{
+    /** @var int */
+    protected $filePerm = 0664;
+    /** @var int */
+    protected $directoryPerm = 0775;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct(array $config)
+    {
+        $this->configurable[] ='directoryPerm';
```

Comment:
It's hard to tell without a stack trace, but that seems unrelated. You sure you didn't remove the class property and then still call it later in the class when getting that error?

Literally all `$configurable` is for is to call setters for config options in [setConfig](https://github.com/thephpleague/flysystem/blob/master/src/Adapter/AbstractFtpAdapter.php#L104-L114).